src/rpc/blockchain.cpp

```diff
@@ -847,17 +847,14 @@ UniValue getchaintips(const UniValue &params, bool fHelp)
             "[\n"
             "  {\n"
             "    \"height\": xxxx,         (numeric) height of the chain tip\n"
+            "    \"chainwork\": \"xxxx\"     (string) total amount of work in this chain, in hexadecimal\n"
             "    \"hash\": \"xxxx\",         (string) block hash of the tip\n"
-            "    \"branchlen\": 0          (numeric) zero for main chain\n"
-            "    \"status\": \"active\"      (string) \"active\" for the main chain\n"
-            "  },\n"
-            "  {\n"
-            "    \"height\": xxxx,\n"
-            "    \"hash\": \"xxxx\",\n"
-            "    \"branchlen\": 1          (numeric) length of branch connecting the tip to the main chain\n"
+            "    \"branchlen\": 0          (numeric) length of branch connecting the tip to the main chain (zero for "
+            "main chain)\n"
             "    \"status\": \"xxxx\"        (string) status of the chain (active, valid-fork, valid-headers, "
```

Comment:
from the nit department: why changing the help for status adding all possible value if later on we are going to list them all with a specific description for every possible value?